### PR TITLE
Screen Time Manager

### PR DESCRIPTION
**Pull request by GaoYeGithub**
# BrowserBuddy Extension Submission

## Project Details

1. **What is the name of your project?**
Screen Time Manager

2. **How many hours did you spend developing this?**  
   _(Give your best estimate)_
7

3. **What problem does your extension solve, or what makes it unique?**  
   _(Provide a brief description of the functionality or innovation)_
Monitor and control your screen time across different websites
Daily Resettable screen limits
Permanently blocked sites to help with self control
Unblock website list
Notifications nearing end of screen time

4. **How many people have tested your extension?**  
   _(We recommend at least 3)_
2 friends and voters on high seas

5. **Does your extension work in both Chrome and Firefox?**  
   _(Your extension must be compatible with both browsers)_
Yep, works on chrome not sure about firefox though.

### CODE REVIEW

File: submissions/ScreenLimits/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Screen Time Manager
```
Comment (JosiasAurel) [inline submissions/ScreenLimits/README.md:1]: image

How does it calculate the time spent? I've left my browser open for more than two minutes including on sites in my blocklist but it never seems to elapse?
It'll be nice to also include instructions on how to use it

File: submissions/ScreenLimits/manifest.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "matches": ["<all_urls>"],
+      "js": ["content.js"]
+    }
+  ],
+  "icons": {
+    "16": "images/icon16.png",
+    "48": "images/icon48.png",
+    "128": "images/icon128.png"
+  }
+}
```
Comment (JosiasAurel) [inline submissions/ScreenLimits/manifest.json:38]: image

It also appears the extension fails to load some icons

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

**Comment by GaoYeGithub** [comment]
Are you using fire fox? I use chrome.storage.sync to check time per minutes 

https://www.loom.com/share/4d85c34d53304777b2e1d5d51a6535b1

**Comment by GaoYeGithub** [comment]
Oops the icons are suppose to be in a folder lemme fix that

**Comment by GaoYeGithub** [comment]
Hmm I did a bit of research and it seems Firefox will still recognize chrome.storage.sync. Is there any errors I can see?

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
image

I'm unable to install the extension on my side. Please test and make sure the extension installs properly on Chrome-based browsers at least

**Comment by GaoYeGithub** [comment]
So sorry I'm unable to reproduce the error, I didn't change any files from the beginning so I'm unsure why your experiencing this.  Maybe try removing and reinstalling? So sorry, I'm very new to chrome extensions

https://www.loom.com/share/c573639347e34f61bd27d0f3f00e53c3

**Review by JosiasAurel — APPROVED** [review]
Awesome! If you haven't yet, please fill [this form](https://airtable.com/appgtKZwZBoOeFU7O/pagyeS51zr2kC1rp0/form) to get your grant.

**Pull request merged**